controllers/admin_client.go

```diff
@@ -22,40 +22,16 @@ package controllers
 
 import (
 	"context"
-	"encoding/json"
 	"fmt"
-	"io/ioutil"
 	"net"
-	"os"
-	"os/exec"
-	"regexp"
-	"strconv"
 	"strings"
 	"sync"
-	"time"
 
 	fdbtypes "github.com/FoundationDB/fdb-kubernetes-operator/api/v1beta1"
 	corev1 "k8s.io/api/core/v1"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-var maxCommandOutput = parseMaxCommandOutput()
-
-var protocolVersionRegex = regexp.MustCompile(`(?m)^protocol (\w+)$`)
-var exclusionLinePattern = regexp.MustCompile("(?m)^ +(.*)$")
-
-func parseMaxCommandOutput() int {
-	flag := os.Getenv("MAX_FDB_CLI_OUTPUT_LENGTH")
-	if flag == "" {
-		return 20
-	}
-	result, err := strconv.Atoi(flag)
-	if err != nil {
-		panic(err)
-	}
-	return result
-}
-
```

Comment:
I think we should also move the `AdminClient interface` implementation into the new package and only keep the mock inside of controllers.

Reply:
That would require the controllers package to take a dependency on the fdbclient package, which means it would take a dependency on the FDB client library.

Reply:
Right, I missed that.